frontend/TestRun/Components/ResultTable.svelte

```diff
@@ -100,37 +195,90 @@
             <div class="table-container">
                 <table class="table table-sm table-bordered">
                     <thead class="thead-dark">
-                    <tr>
-                        <th>
-                            <button class="btn btn-outline-success btn-sm p-1" on:click={copyResultTableAsMarkdown}>
-                                <Fa icon={faMarkdown} size="sm"/>
-                            </button>
-                        </th>
-                        {#each result.columns as col}
-                            <th>{col.name} <span class="unit">{col.unit ? `[${col.unit}]` : ''}</span></th>
-                        {/each}
-                    </tr>
-                    </thead>
-                    <tbody>
-                    {#each result.rows as row}
                         <tr>
-                            <td>{row}</td>
+                            <th>
+                                <button class="btn btn-link p-1" on:click={copyResultTableAsMarkdown}>
+                                    <Fa icon={faMarkdown} size="sm" />
+                                </button>
+                            </th>
                             {#each result.columns as col}
-                                {#key result.table_data[row][col.name]}
-                                    <td class="{ResultCellStatusStyleMap[result.table_data[row][col.name]?.status || 'NULL']}">
-                                        <Cell cell={result.table_data[row][col.name]} bind:selectedScreenshot/>
-                                    </td>
-                                {/key}
+                                <th>
+                                    <div class="column-header">
+                                        <span
+                                            >{col.name}
```

Comment:
that's a weird formatting for html

Reply:
In this case it is justified because `span` is an inline element and adding newline after `>` would cause it to get rendered as an extra space. However since line is short, this can be shortened to 
`<span>{col.name}</span>`.

Reply:
Nevermind, it's a span within a span, I'd say the first span could be omitted

Reply:
additional span removed, I made also whole header cell to be clickable, not just the text